src/AutomatedCar/SystemComponents/CarCollisionDetector.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    using AutomatedCar.Models;
+    using AutomatedCar.SystemComponents.Packets;
+    using Avalonia;
+    using Avalonia.Media;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class CarCollisionDetector : SystemComponent
+    {
+        private CollisionPacket packet;
+        private IEnumerable<WorldObject> collidableWorldObjects;
```

Comment:
We usually name fields with `_` in the beggining. Not something very important but code analysers usually warn you.